frontend/src/metabase/dashboard/actions/data-fetching.js

```diff
@@ -192,7 +191,7 @@ export const fetchDashboard = createAsyncThunk(
         );
         result = {
           ...result,
-          id: IS_EMBED_PREVIEW ? result.id : dashId,
+          id: dashId,
```

Comment:
Revert the fix made in #37972. It's unnecessary to do this now because embed preview parameter values endpoint works the same way as embed endpoint.

Previously, there is no preview embed endpoint for parameter values so we need to call dashboard parameter values instead which takes dashboard ID instead of dashboard token.